module/documents/mixins/pseudo-document.mjs

```diff
@@ -159,7 +159,9 @@ export default Base => class extends Base {
    * @param {RenderOptions} [options]  Rendering options.
    */
   render(options) {
-    for ( const app of this.constructor._apps.get(this.uuid) ?? [] ) app.render(options);
+    for ( const app of this.constructor._apps.get(this.uuid) ?? [] ) {
+      app.render({ window: { title: app.title }, ...options });
```

Comment:
Found a bug?

Reply:
Seems like, the window title doesn't update unless you explicitly pass in `window` to the options (`_updateFrame` checks for that explicitly).